### PR TITLE
GKE custom organizational policies based on CIS Benchmarks

### PR DESCRIPTION
**Pull request by athankso60**
This repository is an attempt to provide sample organizational policies that could be enforced in order to comply with the [GKE portion](https://cloud.google.com/kubernetes-engine/docs/concepts/cis-benchmarks#default_values_on) of CIS benchmarks.

### CODE REVIEW

File: tools/gke-custom-org-policies/run.sh [hunk 1]
```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
```
Comment (shakeebshams) [inline tools/gke-custom-org-policies/run.sh:1]: the script contains arrays and `[[` which are available in bash. The shebang line should be `#!/bin/bash`

File: tools/gke-custom-org-policies/run.sh [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        cd $value
+            content=($(ls -d */))
+            for policy in ${content[@]} 
+            do
+            cd $policy
+                files=($(ls))
+                echo ${files[0]}
+                #run constraint
+                constraint=`cat ${files[0]}`
+                echo "${constraint/"[ORGANIZATIONID]"/$1}"  > constraint.yaml
```
Comment (shakeebshams) [inline tools/gke-custom-org-policies/run.sh:37]: this is another bash line that may not work in a bourne shell

**Review by shakeebshams — COMMENTED** [review]
some suggestions

**Comment by agold-rh** [comment]
Closed as stale. Please re-open if I'm wrong.

**Pull request closed without merging**